webcomponents/bundle/src/main/bundle/grader.properties

```diff
@@ -48,3 +50,7 @@ destroy_rubric_panel_log=Failed to destroy rubric panel. Maybe it wasn't showing
 submission_inline=Inline Submission
 assign_grade_overrides=Assign Grade Overrides
 override_grade_with=Override grade with:
+feedback_instruction = Use the box below to enter additional summary comments about this \
+submission and click Done when you've finished. You still need to save the grade
```

Comment:
Thanks for explaining me what You'll means, I'd never have guessed!

What I meant is you were adding two slightly different versions of the same sentence, and that's still the case:

You still need to save the grade.

vs

You still need to save the grade

Reply:
I wasn't intending to patronise. You are spanish, I am english - I thought that perhaps you thought I'd made a mistake. Many english adults have issues with contractions.

Reply:
Are you talking about the private_notes_tooltip and feedback_instruction?

Reply:
Yes

Reply:
They're separate translations. Are you suggesting that we break them up into smaller strings?

Reply:
They're separate but they're sharing a common bit. I was suggesting either take that shared line to another property, so it can be reused (now it's for 2 but it might be needed on other places too?). Or, at least, use the same words on both cases.

Reply:
Ok, I'll use the same words. Sorry for the confusion on my part.